lib/iris/tests/unit/test_Future.py

```diff
@@ -50,23 +47,43 @@ def test_invalid_attribute(self):
         with self.assertRaises(AttributeError):
             future.numberwang = 7
 
+    def test_cell_datetime_objects(self):
+        future = Future()
+        new_value = not future.cell_datetime_objects
+        with warnings.catch_warnings(record=True) as warn:
+            warnings.simplefilter('always')
+            future.cell_datetime_objects = new_value
+        self.assertEqual(future.cell_datetime_objects, new_value)
+        exp_wmsg = "'Future' property 'cell_datetime_objects' is deprecated"
+        six.assertRegex(self, str(warn[0]), exp_wmsg)
```

Comment:
What is str(warn[0])?

Reply:
It's the single warning message from when the warning was captured as `warn` on L53 cast as a string. This pattern is reasonably common throughout the Iris tests codebase (I copied this pattern!); see, for example, https://github.com/SciTools/iris/blob/master/lib/iris/tests/unit/config/test_NetCDF.py#L43-L51.